src/components/Button.tsx

```diff
@@ -3,7 +3,7 @@ export default function Button({ children, type, disabled }: ButtonProps) {
     <>
       <button
         disabled={disabled}
-        className="bg-blue-500 rounded-xl text-white py-2 px-4 disabled:bg-gray-600"
+        className="bg-blue-500 rounded-xl text-white m-1 py-2 px-4 disabled:bg-gray-600"
```

Comment:
Please remove the margin, we have to add spacing outside the component, maybe with classnames or spacing component

Reply:
Okay, you're right.